module/Photo/src/Photo/Controller/PhotoController.php

```diff
@@ -14,8 +14,35 @@ public function indexAction()
         $albums = $album_service->getAlbums();
         $photo_service = $this->getPhotoService();
 
+        //we'll fix this ugly thing later vv
+        $config = $this->getPhotoService()->getConfig();
+        $basedir = str_replace("public", "", $config['upload_dir']);
+        $albums = $photo_service->populateCoverPhotos($albums);
```

Comment:
Wat doet `populateCoverPhotos()`? Ik neem aan, een random cover photo kiezen, en deze zo in de database instellen?